src/api/services/publishDocuments.js

```diff
@@ -203,8 +204,13 @@ export const publishDocumentsFactory = ({
         ctx.dataset.findLimitFromSkip,
         ctx.publishedDataset.insertBatch,
         versionTransformerDecorator(transformMainResourceDocument),
+        undefined,
+        ctx.job,
     );
-    ctx.job && ctx.job.log('Document published');
+    ctx.job &&
```

Comment:
Is there an actual case when there's no ctx.job? Shouldn't it return from the start if so?